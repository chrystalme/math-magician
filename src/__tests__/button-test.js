import React from 'react';
import renderer from 'react-test-renderer';
import Button from '../components/button';

describe('Button', () => {
  const clickHandler = () => { ''; };
  test('Button snapshot test', () => {
    const component = renderer.create(
      <Button name="AC" clickHandler={clickHandler} />,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
