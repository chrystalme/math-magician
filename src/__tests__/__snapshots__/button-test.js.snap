// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Button snapshot test 1`] = `
<input
  className="m-2 p-3 btn btn-info false false"
  onClick={[Function]}
  type="button"
  value="AC"
/>
`;
